shared/language-specs/languages.ts

```diff
@@ -28,6 +28,14 @@ const clojureSpec: LanguageSpec = {
     commentStyles: [lispStyleComment],
 }
 
+const cobolSpec: LanguageSpec = {
+    languageID: 'cobol',
+    stylized: 'Cobol',
+    fileExts: ['cbl', 'cob', 'cpy', 'dds', 'ss', 'wks', 'pco'],
+    identCharPattern: createIdentCharPattern('-'),
+    commentStyles: [{ lineRegex: /\*/ }],
```

Comment:
question:

The doc for `lineRegex` says:

```
Captures the content of a line comment. Also prevents jump-to-definition
(except when the token appears to refer to code). Python example:
/#\s?(.*)/
```

This includes a capture group for the comment content. But this regex, and also `shellStyleComment` and `cStyleComment` don't include a capture group. Is the doc example wrong, or are both valid?

Reply:
Good question. I'll look into this a bit before merging to make this more clear.